x-pack/test/reporting/functional/reporting.js

```diff
@@ -219,7 +224,30 @@ export default function ({ getService, getPageObjects }) {
           await expectEnabledGenerateReportButton();
         });
 
-        it('generates a report', async () => await expectReportCanBeCreated());
+        it('matches baseline report', async function () {
+          // Generating and then comparing reports can take longer than the default 60s timeout because the comparePngs
+          // function is taking about 15 seconds per comparison in jenkins.
+          this.timeout(180000);
```

Comment:
Should that value also be 36000? If so, should the timeout just be a const at the top of this file to avoid inconsistencies?

Reply:
nah, the visualize snapshot is a lot simpler so the threshold can be much lower.  Actually this makes me think i need to make a couple changes in this PR.  One: use a lower threshold (find out what that needs to be).  Two, use a better visualization and/or time range3.  This is the report:

<img width="954" alt="screen shot 2018-08-01 at 11 44 12 am" src="https://user-images.githubusercontent.com/16563603/43532564-6ad731be-9580-11e8-99e3-bf916b75323b.png">

It would have caught that bug where the edit controls were showing on the left... but this could be much better....